README.md

```diff
@@ -1 +1,81 @@
-# week7-rm-rf-mario
\ No newline at end of file
+# KITTY.IO
+
+#### A CMS (content management system) app that keeps track of the kitty for the guest house.
+
+Find us on [Heroku](#)
+
+Or run locally:
+1. ```git clone https://github.com/FACN1/week7-rm-rf-mario.git
+npm```
+2. ```cd week7-rm-rf-mario.git```
+3. ```npm install```
+4. ```npm run```
```

Comment:
This section is a bit messed up - you only want one opening ``` and one closing ```. Also good to specify the code type - here it would be ```bash